keypad/finite_state_machine/finite_state_machine.py

```diff
@@ -1,8 +1,28 @@
 """Module contains class for the finite state machine"""
 
-from keypad.finite_state_machine.rule import STATES, Rule
+from keypad.finite_state_machine.rule import *
 from keypad.kpc_agent import KPCAgent
 
+RULES = [
+    Rule(STATES.INIT, signal_is_anything, STATES.READ, KPCAgent.init_passcode_entry),
```

Comment:
Does it make sense to use the class itself and not the instance connected to the fsm? Maybe make this a function that takes the instance as an argument, and call this in the fsm init.

Reply:
Also, the `READ`, `READ2`, `READ3` are a bit confusing. Maybe change the names to something a bit more explanatory?